dnfdaemon-server/services/rpm/rpm.cpp

```diff
@@ -225,7 +225,7 @@ libdnf::transaction::TransactionWeakPtr new_db_transaction(libdnf::Base * base,
     if (!comment.empty()) {
         transaction->set_comment(comment);
     }
-    transaction->set_releasever(base->get_vars().get_values().at("releasever"));
+    transaction->set_releasever(base->get_vars().get_value("releasever"));
```

Comment:
Somehow I think that the code requires that the releasever is always available or detected. I think that this is an incorrect assumption. `releasever` is optional and there are user cases where releasever is not needed (e.g. all repositories are defined without $releasever).